app/starship/routines/directPage.js

```
@@ -0,0 +1,173 @@
+                newmessage.setAttribute("id", JSON.parse(res).id)
+                newmessage.innerText = JSON.parse(res).content
+                this.view.querySelector(".chatarea").appendChild(newmessage)
+            }
+            if (shouldstb) {
+                this.view.querySelector(".chatarea").scrollTop = this.view.querySelector(".chatarea").scrollHeight
+            }
+        })
+        var header = this.view.querySelector("qcl-header")
+        var data = await getChannelInfo(this.channelid)
+        var otheruser = data.members.filter(member => member != authsession.data.user.id)[0]
+        var otheruser = await getMember(otheruser)
```

Biome (2.1.2)

[error] Shouldn't redeclare 'otheruser'. Consider to delete it or rename it.

'otheruser' is defined here:


(lint/suspicious/noRedeclare)